### DISCUSSION TITLE
Welcome to keyboards Discussions!

### DISCUSSION THREAD

**Discussion by juhakaup** [juhakaup/keyboards#21] [Announcements]
## 👋 Welcome!
  There seems to be quite a lot of comments and requests coming through issues. 
 However, I feel that It's not the best venue for sharing your ideas and mods.

So I'll open this discussions sections, and we'll see if anyone manages to find it. Personally I didn't know such thing existed until I stumbled upon it a couple days ago.

Cheers, Juha

**Comment by lkhphuc** [comment]
Just FYI, you can also convert existing issues into discussions, so that would help with discoverability.
